in_progress/mission-statement.md

```diff
@@ -0,0 +1,58 @@
+# Mission Statement
+
+## Goals
+
+We want to make single cell omics data analysis as accessible as possible. This means
```

Comment:
```suggestion
We want to make for analysis tools for omics data in the life sciences as accessible as possible. This means
```

Reply:
can I push this change @Zethson @grst ?

Reply:
I honestly have very mixed feelings and will put this onto the agenda for tonight ok?

Reply:
this was discussed, will resolve